openupgradelib/openupgrade_merge_records.py

```diff
@@ -0,0 +1,355 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+import functools
+from psycopg2.extensions import AsIs
+from .openupgrade import logged_query
+from .openupgrade_tools import column_exists, table_exists
+
+logger = logging.getLogger('OpenUpgrade')
+logger.setLevel(logging.DEBUG)
```

Comment:
AFAIK this is called `_logger` as per convention.

Reply:
Well, here in openupgradelib we use `logger`